hivemind_core/protocol.py

```
@@ -671,12 +672,16 @@ def handle_intercom_message(
         pload = message.payload
         if isinstance(pload, dict) and "ciphertext" in pload:
             try:
-                message_from_blob = pgpy.PGPMessage.from_blob(pload["ciphertext"])
+                ciphertext = unhexlify(pload["ciphertext"])
+                signature = unhexlify(pload["signature"])
```

Ruff (0.8.2)

Local variable `signature` is assigned to but never used

Remove assignment to unused variable `signature`

(F841)